README.md

```diff
@@ -68,7 +68,7 @@ requires the full repository to be checked out, not just the server directory.
 
 Server dependencies are listed in `server/requirements.txt`. You can use pip to install
 these dependencies using `pip install -r server/requirements.txt` and/or you can
-use your distribution's package management to fulfill them.
+use your distribution's package management to fulfill them. A redis server is also required.
```

Comment:
Would this also be required if the user doesn't want to use EC2SpotManager at all?

I think if we only use it there, then it should only be required if you actually use that module and in that case, I would like to explicitly state that here. Maybe also mention briefly how this can be installed with Ubuntu as an example.

I know it isn't part of this PR but eventually we should document the same for celery, because FuzzManager without celery is much less preferred by now.

Reply:
Sure. It will still be listed in the `requirements.txt`, but you're right, we're only using this so far in EC2SpotManager, and other modules would work without it.

As I mentioned offline, it also works as a celery message broker. We could switch the default celeryconf.py to use `redis://` instead of `ampq://` which is a much lighter dependency for local server use (redis-server is 168k installed, and runtime is C vs erlang).